owncloudApp/src/main/java/com/owncloud/android/shares/ui/fragment/ShareFileFragment.kt

```diff
@@ -418,7 +422,9 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
             noShares.visibility = View.GONE
             usersList.visibility = View.VISIBLE
             usersList.adapter = userGroupsAdapter
-            setListViewHeightBasedOnChildren(usersList)
+            setListViewHeightBasedOnChildren(
```

Comment:
Why do you refactor it in that way??

Reply:
Hmmm, guess it was refactored when applying project code style.